mettle/common/common.go

```diff
@@ -142,7 +142,7 @@ func checkOutputPaths(paths []string) error {
 
 // CheckPath checks that an individual input or output path doesn't contain any illegal entities.
 func CheckPath(path string) error {
-	if strings.Contains(path, "..") || strings.HasPrefix(path, "/") {
+	if strings.Contains(path, "../") || strings.HasPrefix(path, "/") {
```

Comment:
I'm not sure this is right. What if it was just `..` for example?

Reply:
Ah yup, I guess we can do something cleaner with the `filepath` package. I'll have a look

Reply:
Thanks! I guess we can probably simulate with a few `strings` checks if we have to...